src/integration-test/java/com/commercetools/sync/integration/commons/utils/ProductITUtils.java

```diff
@@ -229,5 +235,38 @@ public static Type createPricesCustomType(
         typeKey, locale, name, ResourceTypeIdsSetBuilder.of().addPrices(), ctpClient);
   }
 
+  /**
+   * Builds a {@link PriceDraft} element
+   *
+   * @param amount the amount to create a price draft.
+   * @param currencyUnits the currency unit of the amount.
+   * @param customerGroupId the customer Group Id to create the reference.
+   * @param validFrom
+   * @param validUntil
+   * @param channelId the channel Id to create the reference.
+   * @param customFieldsDraft the custom fields to set on the resulting price draft.
+   * @return a {@link PriceDraft} element
+   */
+  @Nonnull
+  public static PriceDraft getPriceDraft(
+      @Nonnull final BigDecimal amount,
+      @Nonnull final CurrencyUnit currencyUnits,
+      @Nullable final CountryCode countryCode,
+      @Nullable final String customerGroupId,
+      @Nullable final ZonedDateTime validFrom,
+      @Nullable final ZonedDateTime validUntil,
+      @Nullable final String channelId,
+      @Nullable final CustomFieldsDraft customFieldsDraft) {
+    return PriceDraftBuilder.of(Price.of(amount, currencyUnits))
```

Comment:
Do we have to put `DiscountedPrice` and `Tiers` as well? 
https://docs.commercetools.com/api/projects/products#pricedraft